grades/views.py

```diff
@@ -0,0 +1,44 @@
+from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+from django.forms import inlineformset_factory
+from grades.forms import GradingSystemForm
+from grades.models import GradingSystem, GradeRange
+
+# Create your views here.
+@login_required
+def grading_systems(request):
+    user = request.user
+    default_grading_system = GradingSystem.objects.get(name='default')
+    grading_systems = GradingSystem.objects.filter(creator=user).exclude(
```

Comment:
exclude is not needed, since default is not created by user

Reply:
Needed here because on editing default, while saving the default creator is that user!